tests/TemplateTest.php

```diff
@@ -394,6 +394,11 @@ public static function getGetAttributeTests()
             [true, ['foo' => 'bar'], $arrayAccess, 'vars', [], $anyType],
         ]);
 
+        // test for Closure::__invoke()
+        $closure = fn (): string => 'closure called';
+
+        $tests[] = [true, 'closure called', $closure, '__invoke', [], $anyType];
+
```

Comment:
Now that I see it, maybe we can even inline everything!

```suggestion
        $tests[] = [true, 'closure called', fn (): string => 'closure called', '__invoke', [], $anyType];
```